Geneious/Geneious.download.recipe

```diff
@@ -35,6 +35,19 @@ See http://desktop-links.geneious.com/assets/installers/geneious/GeneiousVersion
 				<string>%MAJOR_VERSION% %RELEASE% %RELEASE%.*(Geneious.*mac.*dmg)</string>
 			</dict>
 		</dict>
+		<dict>
+			<key>Processor</key>
+			<string>RegexReplace</string>
```

Comment:
I believe this needs to be formatted like so:
`<string>com.github.n8felton.shared/RegexReplace</string>`